testing/enclient/enclient.go

```diff
@@ -0,0 +1,156 @@
+package enclient
+
+import (
+	"bytes"
+	"crypto/rand"
+	"encoding/base64"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"math/big"
+	"net/http"
+	"time"
+
+	"github.com/google/exposure-notifications-server/internal/database"
+)
+
+const (
+	dkLen               = 16
+	maxTransmissionRisk = 8
+	maxIntervals        = 144
+	// httpTimeout is the maximum amount of time to wait for a response.
+	httpTimeout = 10 * time.Second
```

Comment:
10 might not be enough w/ request tunneling + cold starts - I've seen a realistic 8 in some circumstances.